SpatialGDK/Extras/schema/authority_intent.schema

```diff
@@ -6,4 +6,4 @@ component AuthorityIntent {
      // Id assigned to the Unreal server worker which should be authoritative for this entity.
      // 0 is reserved as an invalid/unset value.
      uint32 virtual_worker_id = 1;
-}
+}
```

Comment:
Unnecessary whitespace change

Reply:
And in others below

Reply:
I think we should leave this in the interest in getting things shipped ASAP. Let me know if you disagree.